timesheet_activity_report/README.rst

```diff
@@ -15,37 +15,48 @@ Installation
 
 To install this module, you need to:
 
- * have basic modules installed (hr_timesheet, project_timesheet, project_issue_sheet)
-
+#. Have basic modules installed (hr_timesheet, project_timesheet, project_issue_sheet)
```

Comment:
I just copy/pasted the README from another module that also had only 1 item. Any recommendation then?